### PR TITLE
Fix #43: Added postgresql to the database

### LINKED ISSUE #43: Add PostgreSQL
See `Android` in README.md to get an idea of the contents of the PR for this issue.

### PR DESCRIPTION
**Pull request by jonathanalvares9009**
## Explanation

This PR will add PostgreSQL to the database
Fixes #43 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "main" and not on any other branch.
- [x] Have you followed the guidelines in the Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/CSwala/awesome-computer-science/pulls) for the same tech?
- [x] Is your submission is according to the desired pattern ?
- [x] Have you placed the contents in the `Readme.md` keeping the sorting order in mind ?

Does you submission include -

- [x]	Category/Type
- [x]	Official Documnetation  
- [x]	Best YouTube creators  
- [x]	Useful Websites  
- [x]	Dedicated Github page 
- [x]	Best Courses  
- [x]	Ohter Tips / Hacks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -688,6 +688,26 @@ https://www.dataschool.io/future-of-pandas/
...
...e3F9rwKVsaHtOXg1u38mkTL5ifPrDpuP8xtbfiXpzepqwm5jx5A28/6KupL/Cdpc1Tm/oRa1NOE68HA1J4a0Ze73WNGfkTayVZeXPO03YYbeVTd+tDFpM3ulY6nHmr0Lg8bwmO/phJSnw+pgQIgx6x4n00JrkKLFuv40HALX00Z34s0HSVK3y/7JnNkbO3Sv9WZ/iB1taNsG4szUDbYlIzXMG3qm/iwaSNt4UCRTcWbo5qzUEbOp4UyclTukIj6bLT04TdFda5uN6qxt9sd27JBpg7hu048NbeMMg/ksFonh93AYIHgVv+W7UfWVtOLbvoV4zCol9qblrygbgX4rB3vtqP9shKWFJIXuZceHtvxpqRN4mATdabMGKfsNfpUB9VxCIzjO4/kFaGNveiV/aDq5+Eprv3RUx7I3r6F77eJDW+mpwICBWerkS8vNbE6cVOpZz8YjbUqzc76odJlaDqdKCqEYoeDFWqu7/mYZuldtM4qNS7hoiM+cDjoxxBbRexNRPFY7KY+2M6cVpK2Dr47Dko+uTlbnBB5iX9Vx1e31ragg0DjSlj21p1W8LUApvTJoNpSQtHXOtA2owoClodo/+/NXKY5kzTfX1Y4+pPFxpA2Uzn8xzSuGL3kGiCCzbePgNPrqMaDtKfiwUaTgAi4CDgskq7+ZX8ZxdLs5xJO2on8MZ+ZYJLOuN6YHtBm+2yy+DAPaDL+dPoVhTH/jnLUUKENbtjvzRtNo62JD22VrN53kTYWmjHw1BdHvFBwCzY1LBUIb1UyOdMRqsdxpVDsvAW1kelykCG2NS+gRPja1INYdbbHcAWq6HgUjNRUrbXGhrXHVAZnqHUuN0nCKIdti0m1UG90xCpSRH4KK6oPnp16FGmDbGn6g1inlYc6MWULrYyOkV/32frUTVOZFrbQqrV/37f7rWlLl7QFpc2onfITTiL8H4QM7cqZR91vVaMs0Tb/XT8llU0EHW2dYIEqgfam6YQbBB13VHPyDuJqzDrzDPu2P1Pq2KvXJYe7jAzD/e2n7eQTxL6n+YBOf5R72B9e6Gmnv2yMc6csOjVihiTly3nk9yuDL27rb7udz4gez+9J4qh5LCTr877+BblQMvoOPg4ODg4ODg4ODg4ODg4ODg4MjcfgXH81Es9vlQu4AAAAASUVORK5CYII=" width="500" height="500"/>
```
Comment (riturajjain2000) [inline README.md]: remove this image , as its just the logo

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Find if there are any courses on coursera or udacity also

**Comment by jonathanalvares9009** [comment]
Postgresql is not there as a single course on coursera or audacity.

**Comment by jonathanalvares9009** [comment]
@riturajjain2000 Can you review it?

**Review by riturajjain2000 — APPROVED** [review]

**Pull request merged**